<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMedium</class>
 <widget class="QDialog" name="GameMedium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Medium mode</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>467</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mice left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Steps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="8">
       <widget class="QPushButton" name="r0c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QPushButton" name="r2c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="r0c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="r1c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="r2c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="r0c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="r0c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="r2c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="r0c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="r6c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="r6c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="r6c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="r6c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="r6c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="8">
       <widget class="QPushButton" name="r6c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="r7c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="r7c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="r7c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="8">
       <widget class="QPushButton" name="r7c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="7">
       <widget class="QPushButton" name="r7c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QPushButton" name="r6c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="r7c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QPushButton" name="r8c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QPushButton" name="r7c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="r7c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="9">
       <widget class="QPushButton" name="r6c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="9">
       <widget class="QPushButton" name="r7c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="r8c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="9">
       <widget class="QPushButton" name="r8c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="r5c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="r5c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="r6c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="7">
       <widget class="QPushButton" name="r8c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="8">
       <widget class="QPushButton" name="r8c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QPushButton" name="r5c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="9">
       <widget class="QPushButton" name="r5c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="r5c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="r5c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="r7c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="r3c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="r2c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="r4c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="r4c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="r5c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="r4c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="r4c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="r5c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="r5c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="9">
       <widget class="QPushButton" name="r4c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="r4c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="r8c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="r8c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="r8c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QPushButton" name="r8c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="9">
       <widget class="QPushButton" name="r3c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="r1c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="r2c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="r2c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="r3c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="r3c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="r6c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="r8c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="r3c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="r3c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="r3c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="r4c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="r4c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="r3c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QPushButton" name="r3c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="r4c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="r3c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="8">
       <widget class="QPushButton" name="r5c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QPushButton" name="r4c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="r2c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="r1c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="r1c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="r1c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="r1c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="r1c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="r2c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="r2c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="r1c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="r1c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="r0c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="r0c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="r0c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="r0c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="r1c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="r2c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="r0c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="r9c0">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="r9c1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="r9c2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="r9c3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QPushButton" name="r9c4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="5">
       <widget class="QPushButton" name="r9c5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="6">
       <widget class="QPushButton" name="r9c6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="7">
       <widget class="QPushButton" name="r9c7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="8">
       <widget class="QPushButton" name="r9c8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="9">
       <widget class="QPushButton" name="r9c9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>HP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Progress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_101">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_101</sender>
   <signal>clicked()</signal>
   <receiver>GameMedium</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
